u2f-gae-demo/war/origins.json

```diff
@@ -0,0 +1,8 @@
+{
+  "trustedFacets" : [{
+    "version": { "major": 1, "minor" : 0 },
+    "ids": [
+      "android:apk-key-hash:bkHnlWEV_jRCPdYGJfwOl7Sn_CLC_2TE3h4TO1_n34I"
```

Comment:
Web based requests will receive AppId in this form:
[String appId = (req.isSecure() ? "https://" : "http://") + req.getHeader("Host");](https://github.com/google/u2f-ref-code/blob/master/u2f-gae-demo/src/com/google/u2f/gaedemo/servlets/BeginEnrollServlet.java#L46). 
And those request will automatically get AppId authorized due to same origin, so I thought it's not necessary to list those origins there.

This AppId file will only be provided for Android app based requests through endpoints API calls, so adding those u2fdemo appspot origins don't help either.